src/TimingData.cpp

```diff
@@ -1130,18 +1128,6 @@ void TimingData::DeleteRows( int iStartRow, int iRowsToDelete )
 
 float TimingData::GetDisplayedSpeedPercent( float fSongBeat, float fMusicSeconds ) const
 {
-	/* HACK: Somehow we get called into this function when there is no
```

Comment:
Let's keep some form of this comment around. I don't think this condition makes sense since this function isn't static, but keeping this behavior documents in some form sounds good to me.

Reply:
What would you like to keep here? The compiler indicated that `!this` would always evaluate to `false`, so it won't compile if I leave the `if` statement in there.

```
error: ‘nonnull’ argument ‘this’ compared to NULL [-Werror=nonnull-compare]
```

Reply:
Yeah I agree that doing `!this` doesn't make sense. I guess I wanted to document that this case would induce a crash but the behavior really doesn't make sense. I think we can just leave your change as is.